tests/sles4sap/wizard_hana_install.pm

```diff
@@ -157,7 +157,7 @@ sub test_flags {
 sub post_fail_hook {
     my ($self) = @_;
     check_var('BACKEND', 'ipmi') ? use_ssh_serial_console : select_console 'root-console';
-    assert_script_run 'tar cf /tmp/logs.tar /var/adm/autoinstall/logs; xz -9v /tmp/logs.tar';
+    assert_script_run 'tar cf /tmp/logs.tar /var/adm/autoinstall/logs /var/tmp/hdb*; xz -9v /tmp/logs.tar';
```

Comment:
This will fail when `/var/tmp/hdb*` is not present and prevent the logs from being uploaded. Are you sure the path `/var/tmp/hdb*` is always present? I suppose it will not be there if the test fails before the wizard is executed.

Reply:
It will not fail because we're using `;` and not `&&`.

Reply:
You're correct.